### PR TITLE
apps/engine: add EC to list of capabilities

### PR DESCRIPTION
**Pull request by iomartin**
openssl engine -c wasn't showing if an engine implemented EC

cla: trivial

### CODE REVIEW

**Review by t8m — APPROVED** [review]
OK with CLA: trivial

**Review by shahsb — APPROVED** [review]

**Comment by iomartin** [comment]
Hi @t8m, is there anything else I can do to get this PR merged? I believe the CI failure is unrelated to my PR.

**Comment by t8m** [comment]
> Hi @t8m, is there anything else I can do to get this PR merged? I believe the CI failure is unrelated to my PR.

Nope, the ball is on our side now. It needs a second approval from @openssl/committers

**Review by romen — APPROVED** [review]
LGTM and agree on trivial

**Comment by t8m** [comment]
Merged to the master branch. Thank you for the contribution.

**Pull request closed without merging**
